remote/remote_state_s3.go

```diff
@@ -15,12 +15,12 @@ import (
 
 // A representation of the configuration options available for S3 remote state
 type RemoteStateConfigS3 struct {
-	Encrypt   bool   `mapstructure:"encrypt"`
-	Bucket    string `mapstructure:"bucket"`
-	Key       string `mapstructure:"key"`
-	Region    string `mapstructure:"region"`
-	Profile   string `mapstructure:"profile"`
-	LockTable string `mapstructure:"lock_table"`
+	Encrypt       bool   `mapstructure:"encrypt"`
+	Bucket        string `mapstructure:"bucket"`
+	Key           string `mapstructure:"key"`
+	Region        string `mapstructure:"region"`
+	Profile       string `mapstructure:"profile"`
+	DynamoDBTable string `mapstructure:"dynamodb_table"`
```

Comment:
I'm a little concerned about dropping the old param. Ideally, we'd follow Terraform's lead and report a deprecation warning for at least a little while. Or, even simpler, Terragrunt should accept both params and pass them on to Terraform unchanged, and Terraform itself would log the deprecation warning.

Reply:
Yeah, I agree, but that's more work :)

I'll see if I can make some time to figure that out soon.

R.

Reply:
not a go expert but can't you just keep both params?

```go
LockTable string `mapstructure:"lock_table"`
DynamoDBTable string `mapstructure:"dynamodb_table`"
```

Reply:
Yup, keeping both is most likely the way to go. Also need to check both in the `createLockTableIfNecessary` method.